src/ui/config/config_test.go

```diff
@@ -117,6 +117,10 @@ func TestConfig(t *testing.T) {
 	if _, err := Database(); err != nil {
 		t.Fatalf("failed to get database: %v", err)
 	}
+
+	if _, err := ClairDB(); err != nil {
```

Comment:
This is trivial but it would be better if you can verify the content of the DB config in test case.